cmd/getbinariesfromquay/main.go

```diff
@@ -114,14 +114,15 @@ func copyBinaryFromLastLayer(src, dest, binaryPath string) error {
 	if err != nil {
 		return err
 	}
-	last := man.LayerInfos()[len(man.LayerInfos())-1]
-	layerTar, err := os.Open(path.Join(src, last.Digest.String()[7:]))
-	if err != nil {
-		return err
-	}
-	err = tar.Untar(src, layerTar)
-	if err != nil {
-		return err
+	for _, l := range man.LayerInfos() {
+		layerTar, err := os.Open(path.Join(src, l.Digest.String()[7:]))
+		if err != nil {
+			return err
+		}
+		err = tar.Untar(src, layerTar)
```

Comment:
Is this really good enough? This won't correctly handle whiteout files in the layers. :thinking: but I guess thats fine since presumably CMD won't point to a deleted file.